src/client/client.ts

```diff
@@ -148,6 +221,109 @@ export default class CTProtoClient<AuthRequestPayload, AuthResponsePayload, ApiR
     this.init();
   }
 
+  /**
+   * This method sends file requests
+   *
+   * @param type - available type of requests
```

Comment:
I think, the type can be specified internally by this method. Users shouldn't care about it:

```js
client.sendFile(file, { fileName: fileName })
```

Reply:
I think, that type needs to be here, because it is more comfortable to handle file uploading on the server side